src/index-template.ejs

```diff
@@ -9,8 +9,6 @@
   <meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
   <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
 
-  <title><%= htmlWebpackPlugin.options.title %></title>
```

Comment:
Jeg har ikke fått testet, men er du sikker på at helmet erstatter title her?

Reply:
Helmet funker fint i yarn start og yarn build+serve sånn siste commit er i hvert fall :)

Reply:
Hm, jeg ser at title og description dukker opp etter at javascript er lastet og react-helmet har kjørt, men de finnes ikke i html-filene. Hvordan er det da med SEO og indeksering av google og andre søkemotorer? Får de med seg title og description?

Reply:
index-template.ejs brukes vel bare i yarn start? Har fikset for renderStatic :)

Reply:
ok